great-idea/contact.html

```diff
@@ -0,0 +1,100 @@
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/index.css" />
+
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+  </head>
+
+  <body>
+    <div class="container">
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="contact.html">Contact</a>
+
+        <a href="index.html"
+          ><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."
+        /></a>
+      </nav>
+
+      <div class="content">
+        <div class="centerer">
+          <img
+            class="contact-header-img"
+            src="img/contact.png"
+            alt="Contact us header image"
+          />
+        </div>
+
+        <h2>Contact Us</h2>
+
+        <p>
+          If you have any question please contact us. Proin vitae tellus rutrum, accumsan nunc sit amet,
+          iaculis tortor. Mauris consectetur feugiat justo quis aliquet.
+          Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed
+          faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.
+        </p>
+
+        <hr />
+
+        <div class="cards">
+          <div class="card">
+            <h3>Social Media</h3>
+
+            <p>
+              Nulla suscipit interdum diam at maximus. Proin vitae tellus
+              rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur
+              feugiat justo quis aliquet. Curabitur cursus et justo eu
+              consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+              eleifend. Mauris euismod facilisis iaculis.
+            </p>
+          </div>
+
+          <div class="card">
+            <h3>Email</h3>
+
+            <p>
+              Nulla suscipit interdum diam at maximus. Proin vitae tellus
+              rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur
+              feugiat justo quis aliquet. Curabitur cursus et justo eu
+              consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+              eleifend. Mauris euismod facilisis iaculis.
+            </p>
+          </div>
+
+        </div>
+
+        <hr />
+
+        <div class="map-container">
+          <div class="map">
+            <h3>Where are we?</h3>
+
+            <div class="mapouter"><div class="gmap_canvas"><iframe width="600" height="500" id="gmap_canvas" src="https://maps.google.com/maps?q=boulevard%20avenue%2C%20ny&t=&z=13&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>Werbung: <a href="https://www.pureblack.de">pureblack.de</a></div><style>.mapouter{position:relative;text-align:right;height:500px;width:600px;}.gmap_canvas {overflow:hidden;background:none!important;height:500px;width:600px;}</style></div>
```

Comment:
I'm impressed with the addition of the map in your project. Very well done!
This line is too long though, you should break it in multiple lines for added readability.